convert2rhel/utils.py

```diff
@@ -533,3 +533,13 @@ def backup(self):
 
 
 changed_pkgs_control = ChangedRPMPackagesController()  # pylint: disable=C0103
+
+
+def get_os_release():
+    """Parse /etc/os-release into dict."""
+    os_release = {}
+    with open("/etc/os-release") as f:
```

Comment:
```suggestion
    with open("/etc/system-release") as f:
```

`os-release` is not on CentOS/OL/RHEL 6. `system-release` is on all major versions - 6, 7, 8.